wsmaster/che-core-api-factory/src/main/java/org/eclipse/che/api/factory/server/scm/AuthorizingFileContentProvider.java

```diff
@@ -48,26 +52,41 @@ public AuthorizingFileContentProvider(
 
   @Override
   public String fetchContent(String fileURL) throws IOException, DevfileException {
-    return fetchContent(fileURL, false);
+    return fetchContent(fileURL, false, null);
+  }
+
+  @Override
+  public String fetchContent(String fileURL, String credentials)
+      throws IOException, DevfileException {
+    return fetchContent(fileURL, false, credentials);
   }
```

Comment:
can we use Optional here also?

Reply:
There is no need to use `Optional` or `Nullable` here. This is an overloaded method which is supposed to receive non null parameters.